recipe/meta.yaml

```diff
@@ -64,6 +65,9 @@ test:
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}
```

Comment:
Ah we might also want to update this for 11

Reply:
But we actually need nvcc to test compiling, so I am confused by why keeping nvcc here is not accompanied by a correct cudatoolkit...